### ISSUE TITLE
Partial view depth 2 throws 404

### ISSUE THREAD

**Issue by Abdjulaziz** [umbraco/Umbraco.Cloud.Issues#806]
## Partial view depth 2 throws 404
Files that are two levels deep in any folder under templates ↓ trow `ApiError Not Found`
![image](https://github.com/user-attachments/assets/2a67c2d3-5679-4e76-a247-2391a4392e21)
![image](https://github.com/user-attachments/assets/f64e9a03-3425-4fb3-a382-2d42791ce210)

I've had the issue with:
14.0.0
14.1.0
14.1.1

The random thing is it throws an error it cannot access the files and create files but it is able to create files but they are not accessible and give this error: 
![image](https://github.com/user-attachments/assets/6f3e685c-5e4f-4205-bc6f-b489c8459147)

### Issue description

related to this issue, that should have been fixed but still presists on cloud: https://github.com/umbraco/Umbraco-CMS/issues/16249

**Comment by mikkelhm** [comment]
Hey @Abdjulaziz What version of Umbraco did you try this out in ? was it a 14.0 or the 14.1 version?

**Comment by Abdjulaziz** [comment]
14.0.0
14.1.0
14.1.1

**Comment by lars-lindstrom** [comment]
I can add
14.2.0--rc.preview.9.g7e444e5

**Comment by Abdjulaziz** [comment]
Still an issue on 14.3.0

**Comment by j9850s** [comment]
This also happens on stylesheets and scripts within folders.
This is because umbraco is using an illigal character in the path. (%2F). When the host is configured for correcting this, umbraco fails. (for example Azure WebApps are correcting this). But the raw `PathAndQuery` is placed in the `X-Waws-Unencoded-Url.`)

A Host sulution is adding middleware for restoring the 'raw' path value.
Otherwise, umbraco should encode the {path} part or placing the parameter outside the path and in the querystring

**Comment by j9850s** [comment]
Fix for umbraco-team. (i cant push the code)

Fix these files:
```
ByPathPartialViewFolderController.cs
ByPathPartialViewController.cs
ByPathScriptFolderController.cs
ByPathScriptController.cs
ByPathStylesheetFolderController.cs
ByPathStylesheetController.cs
```

Change the `[HttpGet("{path}")]` attribuut into `[HttpGet("{*path}")]` (see the asterix difference)

This was also done in the past for the `ByPathMediaApiController`

**Comment by Abdjulaziz** [comment]
Fixed in v15 after the this implementation: [#17468](https://github.com/umbraco/Umbraco-CMS/pull/17468)
Next v14 upgrade should also have the fix 😄
